charts/pega/values.yaml

```diff
@@ -231,10 +232,18 @@ installer:
   # Upgrade specific properties
   upgrade:
     # Type of upgrade
-    # Valid values are 'in-place' , 'out-of-place'
+    # For Pega Platform 8.3 and later, to complete a zero-downtime patch, specify 'out-of-place';
+    # For installations, leave this area set to the default values.
     upgradeType: "in-place"
-    # Specify target rules schema for migration and upgrade
+    # and data schemas required for migration and upgrade; for 8.3.x and later, the deployment 
+    # automatically creates and names the schemas with the name you specify.
+    # For deployments running Pega Infinity 8.2.1 through 8.2.7 only: You must create a new
+    # blank, target rules schema in your existing database; leave this new schema empty.
+    # Specify the target Rules schema name and leave the target Data schema parameter empty.
     targetRulesSchema: ""
+    # commenting out the new, required parameter to ensure the PR builds and can be reviewed.
+    # Saurabh confirmed that this parameter is required for patch process to work correctly.
+    #targetDataSchema: ""
```

Comment:
Your PR is failing the travis build because of some persnickety rules about whitespace in the comment on this line.

You should change it to read:
```
      # targetDataSchema: ""
```
(note the space between '#' and 'targetDataSchema')

Reply:
persnickety inDEED! thanks for the advice. I really appreciate it.